sdk/cpp/core/src/netconf_client.hpp

```diff
@@ -38,7 +39,7 @@ typedef struct capabilities {
 namespace ydk
 {
 
-class NetconfClient
+class NetconfClient : public NetconfClientBase
```

Comment:
To avoid future confusion, can we change names as below?
```
class NetconfSSHClient : public NetconfClient
```

and
```
class NetconfTCPClient : public NetconfClient
```